### PR TITLE
Posts/moving to GitHub

### PR DESCRIPTION
**Pull request by borsi**
Pls comment and review.

### CODE REVIEW

**Comment by balazskeszei** [comment]
Title: add after the current title: "Sensenet ECM is on GitHub now."

**Comment by herflis** [comment]
Change taglist to:

`tags: [git, github, stack overflow, codeplex, gitter, community, blog]`

so that the 'stack overflow' tag could be indentified as one

File: _drafts/2017-05-23-moving-to-github.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+We moved. To **Github**, with our code and management tools… and our documentation **and** this blog(!); and to Stack Overflow with your questions and our answers.
+
+---
+
+Finally. Okay, we've been moving to these platforms for almost a year, but still. We haven't yet had the chance to tell anyone. Also, I think this is the stage when one can look around and say that - in spite of the several unopened boxes and the fact that the paint is not yet dried on the wall - sensenet ECM is finally home again.
+
+So why did we suddenly drop Codeplex and our own Forum (and blog) in favor of external services?
+
+## Visibility,  visibility, visibility
+We can be [the best .NET Open Source Enterprise Content Management System in the Universe][e68b21b6] if noone knows about it, right? And the World has moved a lot since we set up the tools we were using since.
```
Comment (iviczl) [inline _drafts/2017-05-23-moving-to-github.md]: ...we have been using since.

File: _drafts/2017-05-23-moving-to-github.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  [e68b21b6]: https://www.sensenet.com/product "sensenet ECM"
+
+## GitHub is the de-facto place for open source
+It just really makes [sense][00c66cf2] for us to be there as well. This way, we can focus on development and on delivering the most value for everyone involved. We'd like to continue giving a lot of value to our Community (and even more to our Enterprise customers) so you'll see some buzz near our repositories.
+
+  [00c66cf2]: https://www.sensenet.com/community "sensenet Community"
+
+## Increased productivity
+Over the past years we used a plethora of tools for managing our development process. Now - almost - everything is on GitHub. In the last 3 sprints we tried out the[ ZenHub chrome extension][03b7e478] and so far, it's great. Before this we had a separate backlog, we had our issues in Jira (to be fair, for our [enterprise customers][09dba056], we still do) and we had several repositories in TFS. It was a convoluted process to put together a decent release and even more complicated to get feedback from our users. Now you can:
```
Comment (iviczl) [inline _drafts/2017-05-23-moving-to-github.md]: Now, almost everything is ...

**Review by balazskeszei — APPROVED** [review]

**Pull request merged**